### ISSUE TITLE
New action:  Invoke Mission Control, and immediately show desktop preview.

### ISSUE THREAD

**Issue by Detnator-Neo** [rxhanson/Multitouch-Community#189]
When Mission Control is activated (via any means available -- opening the Mission Control app, a gesture set in Trackpad/Mouse Settings, hot corner, whatever) it spreads all the windows out, and shows the bar at the top with the different spaces and desktops, but it's a minimal version of that bar at that point.  Then you move the mouse up to that bar and it expands down showing all the spaces and desktops in more detail and allowing you to move them around and whatnot.

Just feels like an unnecessary extra step to have to separately move the mouse up to that area to see the expanded version of that spaces and desktops bar.

Better Touch Tool (BTT) has an option to "Mission Control, and immediately show desktop preview".  I've set my Mac up to use that for a three finger up swipe. I'd like to replace BTT with Multitouch.app, but this is one potential deal breaker.  What do you think?

**Comment by rxhanson** [comment]
Makes sense to me. I'll add this to my backlog, but it as noted in other issues, it will be a while before I will get back to a "new features" release on Multitouch.

**Comment by Detnator-Neo** [comment]
Fair enough.  I'll look forward to it when it happens.  

It occurs to me I could write to KM's author and see if he'll add it to KM.  That would solve it for me also. 😊   But if not, I'll admit of all my requests, this one's the one that could stop me replacing BTT with it.  Everything else I can workaround (multiple actions, KM integration via bash, etc) but BTT is the only app I know of currently that does this.  

Might seem like a small thing, but for some reason I just find it really frustrating that that desktop preview bar at the top doesn't show automatically by default.  There's almost never a time I want Mission Control without the full desktop preview bar.  Maybe it's just my OCD... 🥴🤓
